community-guidelines.md

```diff
@@ -0,0 +1,47 @@
+# Polyhack Community Guidelines
+
+**[#polyhack](https://github.com/polyhack)** is a community of
+programmers based on the IRC freenode network. It is specifically a
+polyglot group concerned to discuss programming ideas over the long-term.
+
+
+## Specific Guidelines
+
+ - Your guess is as good as mine
```

Comment:
This list is half the point of this PR. I could dump a bunch of items in here based on what the current code of conduct seems to be, or what people seem to think it should be, but I think it would be much better if everyone put those things down in their own words. It is all just a starting point for discussion anyway.

I guess I'll give it a week for people to add items, and if this is still empty I will make some time on the weekend to dredge through and try to infer something that in some way resembles what everyone seems to think.